packages/web-components/fast-foundation/src/data-grid/data-grid.spec.md

```diff
@@ -70,6 +67,99 @@ Note: using a custom `gridColumn` property allows authors to place cells anywher
+
+- Page Up: Moves focus up an author-determined number of rows, typically scrolling so the top row in the currently visible set of rows becomes one of the last visible rows. If focus is in the first row of the grid, focus does not move.
+
+- Home: Moves focus to the first cell in the row that contains focus.
+
+- End: Moves focus to the last cell in the row that contains focus.
+
+- Control + Home: Moves focus to the first cell in the first row.
+
+- Control + End: Moves focus to the last cell in the last row.
+
+Further, authors can enable navigation within a cell as described [here](https://w3c.github.io/aria-practices/#gridNav_inside) using the `ColumnDefinition` for that cell's column.
+
+For example given a nested grid scenario like this:
+
+```
+<fast-data-grid id="nestedGrid" grid-template-columns="1fr 1fr" generate-header="none">
+    <fast-data-grid-row>
+        <fast-data-grid-cell grid-column="1">1.1</fast-data-grid-cell>
+        <fast-data-grid-cell grid-column="2">
+            <fast-data-grid
+                grid-template-columns="1fr 1fr"
+                generate-header="none"
+                no-tabbing="true"
+            >
+                <fast-data-grid-row>
+                    <fast-data-grid-cell grid-column="1">1.2.1.1</fast-data-grid-cell>
+                    <fast-data-grid-cell grid-column="2">1.2.1.2</fast-data-grid-cell>
+                </fast-data-grid-row>
+                <fast-data-grid-row>
+                    <fast-data-grid-cell grid-column="1">1.2.2.1</fast-data-grid-cell>
+                    <fast-data-grid-cell grid-column="2">1.2.2.2</fast-data-grid-cell>
+                </fast-data-grid-row>
+            </fast-data-grid>
+        </fast-data-grid-cell>
+    </fast-data-grid-row>
+</fast-data-grid>
+```
+
+An author could define a `ColumnDefinition` that specifies that the cells in that column have an internal focus queue and a callback that species where focus goes inside the cell: 
+
+```
+function getFocusTarget(cell: DataGridCell): HTMLElement {
+    return cell.children[0] as HTMLElement;
+}
+const nestedColumn: ColumnDefinition = {
+    ...
+    cellInternalFocusQueue: true,
+    cellFocusTargetCallback: getFocusTarget,
+    ...
+};
+```
+
+### Grid Selection
+
+Selection of cells and rows within the grid can be enabled via the grid's `selection-mode` attribute.  It is set to "none" by default.  
+
+Keyboard selection model is based on guidance [here](https://w3c.github.io/aria-practices/#keyboard-interaction-for-data-grids) when applicable.
+
+#### "single-row" selection mode: 
+
+When in single row selection mode a maximum of one row can be selected at a time.  All rows are labelled with `aria-selected' with any selected row having a value of "true".
+When the grid's `click-select` attribute is set to "true", which is the default, any click on a selectable row will select that row, or deselect if already selected.
```

Comment:
Do we need this to be "click-selected" or should this just be "selected"? I'm not sure I get the difference between the two!

Reply:
"click-select" refers to whether the grid row changes selection state with a click anywhere in the row, not the actual selection state.  An author who sets "click-select" to false would typically be doing so in order to implement their own ui within the row that toggles selection, like for a example a checkbox in a single cell.

Reply:
This has changed significantly and is now covered by "selection-behavior" attribute